File: PlayerBuild/RaidStats.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Newtonsoft.Json.Linq;
using TT2Advisor.Import;

namespace TT2Advisor.PlayerBuild
{
    public class RaidStats
    {
        //public RaidStats(JToken raidStatsToken)
        //{
        //    RaidLevel = (int)raidStatsToken.SelectToken("['Raid Level']");
        //    RaidDamage = (int)raidStatsToken.SelectToken("['Raid Damage']");
        //    TotalRaidExperience = (int)raidStatsToken.SelectToken("['Total Raid Experience']");
        //    TotalRaidAttacks = (int)raidStatsToken.SelectToken("['Total Raid Attacks']");
        //    TotalRaidCardLevels = (int)raidStatsToken.SelectToken("['Total Raid Card Levels']");
        //    RaidCardsOwned = (int)raidStatsToken.SelectToken("['Raid Cards Owned']");
        //    LifetimeClanMorale = (int)raidStatsToken.SelectToken("['Lifetime Clan Morale']");
        //}

        public RaidStats(Import.RaidStats raidStats)
        {
            RaidLevel = raidStats.RaidLevel.ToInt();
            RaidDamage = raidStats.RaidDamage.ToInt();
            TotalRaidExperience = raidStats.TotalRaidExperience.ToInt();
            TotalRaidAttacks = raidStats.TotalRaidAttacks.ToInt();
            TotalRaidCardLevels = raidStats.TotalRaidCardLevels.ToInt();
            RaidCardsOwned = raidStats.RaidCardsOwned.ToInt();
            LifetimeClanMorale = raidStats.LifetimeClanMorale.ToInt();
        }

        public int RaidLevel { get; set; }
        public int RaidDamage { get; set; }
        public int TotalRaidExperience { get; set; }
        public int TotalRaidAttacks { get; set; }
        public int TotalRaidCardLevels { get; set; }
        public int RaidCardsOwned { get; set; }
        public int LifetimeClanMorale { get; set; }
    }
}

File: TT2Advisor.UI/ClientApp/src/components/myBuild/raidStats.js
import React, { Component } from 'react';

export class RaidStats extends Component {

    raidStats: any;

    constructor(props) {
        super(props);
        this.state = { raidStats: this.props.raidStats, loading: true };
    }

    componentDidMount() {

    }

    render() {
        let contents = this.state.loading
            ? <p><em>Loading...</em></p>
            : RaidStats.renderRaidStats(this.state.raidStats);

        return (
            <div>
                <h2>Raid Stats</h2>
                <table className='table table-striped' aria-labelledby="tabelLabel">
                    <thead>
                        <tr>
                            <th>Stat</th>
                            <th>Value</th>
                        </tr>
                    </thead>
                    <tbody>
                        <tr>
                            <td>Raid Level</td>
                            <td>{this.state.raidStats.raidLevel}</td>
                        </tr>
                        <tr>
                            <td>Raid Damage</td>
                            <td>{this.state.raidStats.raidDamage}</td>
                        </tr>
                        <tr>
                            <td>Total Raid Experience</td>
                            <td>{this.state.raidStats.totalRaidExperience}</td>
                        </tr>
                        <tr>
                            <td>Total Raid Attacks</td>
                            <td>{this.state.raidStats.totalRaidAttacks}</td>
                        </tr>
                        <tr>
                            <td>Total Raid Card Levels</td>
                            <td>{this.state.raidStats.totalRaidCardLevels}</td>
                        </tr>
                        <tr>
                            <td>Raid Cards Owned</td>
                            <td>{this.state.raidStats.raidCardsOwned}</td>
                        </tr>
                        <tr>
                            <td>Lifetime Clan Morale</td>
                            <td>{this.state.raidStats.lifetimeClanMorale}</td>
                        </tr>
                    </tbody>
                </table>
            </div>
        );
    }
}
